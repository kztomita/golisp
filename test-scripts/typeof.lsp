(print (type-of nil))
(print (type-of t))
(print (type-of 10))
(print (type-of 10.1))
(print (type-of 'foo))
(print (type-of "foo"))
(print (type-of (lambda ())))
