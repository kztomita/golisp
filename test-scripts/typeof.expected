
null 
boolean 
integer 
float 
symbol 
keyword 
string 
function 
